### PR TITLE
GUI 1605 - When security group field gets cleared on launch config and instance launch wizards, set securityGroups to empty, []

### PR DESCRIPTION
**Pull request by kyolee310**
https://eucalyptus.atlassian.net/browse/GUI-1605

### CODE REVIEW

File: eucaconsole/static/js/pages/instance_launch.js [hunk 1]
```diff
@@ -77,6 +77,9 @@ angular.module('LaunchInstance', ['TagEditor', 'BlockDeviceMappingEditor', 'Imag
             $('#securitygroup').chosen({'width': '100%', search_contains: true});
         };
         $scope.updateSelectedSecurityGroupRules = function () {
+            if ($scope.securityGroups == undefined) {
```
Comment (kamalgill) [inline eucaconsole/static/js/pages/instance_launch.js]: @kyolee310: need a '===' here (and at line 109 below) to pass JSHint

Comment (kyolee310) [inline eucaconsole/static/js/pages/instance_launch.js]: Updated to ===

**Comment by euca-qa-gui-tester** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com:8080/job/user-console-regression-sequence//job/user-console-PR-monitor/35/
Test PASSed.

**Comment by euca-qa-gui-tester** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com:8080/job/user-console-regression-sequence//job/user-console-PR-monitor/36/
Test FAILed.

**Comment by kamalgill** [comment]
Review of this PR is blocked by broken PR monitor status links and a missing "Details" link for the in-progress build.

**Comment by kyolee310** [comment]
@aliceh See the comment above.

**Comment by aliceh** [comment]
I reviewed the test, it's good to merge 

http://jenkins.qa1.eucalyptus-systems.com/job/user-console-regression-sequence/13734/

**Comment by aliceh** [comment]
@kamalgill the build link is broken because it is from 5 days ago. I fixed the links since then. You can re-run the test as needed with the usual comment.

**Pull request merged**